### DISCUSSION TITLE
Variable rates depending on hour of the day.

### DISCUSSION THREAD

**Discussion by Tiritibambix** [kimai/kimai#3268] [Questions & Answers] [answered]
Hello !

Just managed to install Kimai and it looks promising.

I'm trying to set an instance up for the company I'm working in.

The thing is that we have different hourly rate depending on the hours of the day.

We're paid let's say 30€/h from 9:00AM to 8PM, 35€ from 8PM to 00:00 and 39€ from 00:00 and beyond.

How can I reflect that in Kimai ?

**Comment by kevinpapst** [comment, marked as answer]
You can't right now. 
If your company wants to sponsor such a feature, please get in touch.
